helper/identity/templating.go

```diff
@@ -0,0 +1,203 @@
+package identity
+
+import (
+	"errors"
+	"strings"
+)
+
+var (
+	UnbalancedTemplatingCharacterErr = errors.New("unbalanced templating characters")
```

Comment:
Either remove the Err suffix here, or make them all the same (preferably "Err...")